### ISSUE TITLE
iOS Web Incompatibility

### ISSUE THREAD

**Issue by Joeao** [microsoft/NoSQLProvider#74]
Hi team,

WebSQL support was dropped in [Safari 13](https://developer.apple.com/documentation/safari_release_notes/safari_13_release_notes). I'd assumed that my app was using IndexedDB on web iOS devices as my provider order was IndexedDB first, however that isn't compatible.

LocalDB's output on both Safari and Chrome on iOS 13.3:
```
// 'nosqlprovider/dist/IndexedDbProvider'
Provider 0: Open Failure: "Safari doesn't properly implement IndexedDB"
// 'nosqlprovider/dist/WebSqlProvider'
Provider 1: Open Failure: "No support for WebSQL in this browser"
```

You can replicate this in the [Todo List Example](https://github.com/microsoft/reactxp/tree/master/samples/TodoList).

In Memory provider works, but that's not ideal due to lack of persistence.

The workaround for the time being is to re-enable WebSQL on iOS (Settings > Safari > Advanced > Disable ‘Disable Web SQL’), but I assume that will only work for so long.

I just wanted to bring this to your attention as nosqlprovider was working well with this setup, but no longer is due to Apple's decision to remove WebSQL support.

From my limited information, it seems as though Safari on MacOS works, but I don't have the means to confirm.

I appreciate your hard work in everything related to ReactXP.

**Comment by deregtd** [comment]
Oh, blah.  It looks like indexeddb support on safari 13 is at least up to snuff, so we can change the default there.  Thanks for the report.

**Comment by deregtd** [comment]
@Joeao I don't have a test case easily at hand right now, can you possibly check that this branch fixes your cases? If not I can just try releasing a new build version for you to try.

https://github.com/microsoft/NoSQLProvider/tree/ddr/indexeddbfilter

**Comment by deregtd** [comment]
Oops somehow I messed up the branches.  In any event, I spent some time bumping package versions for builds too, and ended up with this PR:
https://github.com/microsoft/NoSQLProvider/pull/75

**Comment by Joeao** [comment]
@deregtd Yep, https://github.com/microsoft/NoSQLProvider/pull/75 is showing:
```
// 'nosqlprovider/dist/IndexedDbProvider'
Provider 0: Open Success!
```

Thanks for sorting it so quickly!

**Comment by deregtd** [comment]
Great!  I'll wait for @berickson1 to take a gander at it and then release a new version.

**Comment by deregtd** [comment]
Published 0.6.23.  Sorry that took so long, didn't realize Brent had accepted the PR!
